ts/app.ts

```diff
@@ -22,24 +24,35 @@ if (data) {
   id = 0;
 }
 
-function loadList(array) {
-  array.forEach(function (item) {
+function loadList(
+  array: { name: string; id: number; done: boolean; trash: boolean }[]
+) {
+  array.forEach(function (item: {
+    name: string;
+    id: number;
+    done: boolean;
+    trash: boolean;
+  }) {
     addToDo(item.name, item.id, item.done, item.trash);
   });
 }
 
-clear.addEventListener("click", function () {
+clear!.addEventListener("click", function () {
   localStorage.clear();
   location.reload();
 });
```

Comment:
for all `addEventListener`: export the functions to an `utils.ts` file and reference it here. Move all event listeners to the bottom of the file so it loads everything first and then applies them to the DOM